cogs/customs.py

```diff
@@ -40,70 +30,77 @@ def __init__(self, bot):
+                 "activity, and unfortunately that info is listed in the same window as the \"prune\" function. "
+                 "A mis-click occurred and *quite* a few of you were boop'd. We all know tech derps sometimes "
+                 "happen, but we want to once more sincerely apologize and hope no one was offended or hurt by it.")
         await ctx.channel.send(embed=embed)
 
     @commands.command()
     @commands.is_owner()
     @commands.guild_only()
     async def kicked(self, ctx):
-        connection = self.config.db_connection()
-        cursor = connection.cursor()
-        cursor.execute("""SELECT DISTINCT `id_user` FROM `statistics_global`""")
-        member_ids = cursor.fetchall()
-        nb_kicked = 0
-        for member_id in member_ids:
-            member = ctx.guild.get_member(int(member_id[0]))
-            if member is None:
-                cursor.execute("""INSERT INTO `kicked` (`id`, `id_user`) VALUES (NULL, %s)""", member_id[0])
-                connection.commit()
-                nb_kicked += 1
-        connection.close()
+        with self.cursor_context(commit=True) as cursor:
+            cursor.execute("""SELECT DISTINCT `id_user` FROM `statistics_global`""")
+            member_ids = cursor.fetchall()
+            nb_kicked = 0
+            for member_id in member_ids:
+                member = ctx.guild.get_member(int(member_id[0]))
+                if member is None:
+                    cursor.execute("""INSERT INTO `kicked` (`id`, `id_user`) VALUES (NULL, %s)""", member_id[0])
+                    nb_kicked += 1
+
         await ctx.author.send('{} Users were kicked'.format(nb_kicked))
 
     @commands.command()
     @commands.is_owner()
     @commands.guild_only()
     async def safeguard(self, ctx):
         # Get "User" role
-        user_role = None
-        for role in ctx.guild.roles:
-            if role.name == "Users":
-                user_role = role
+        user_role = discord.utils.get(ctx.guild.roles, name="Users")
         if user_role is None:
             await ctx.author.send("There is no Users role !")
-            return False
+            return
+
         nb_user = 0
         nb_bot = 0
         nb_has_role = 0
-        message = await ctx.channel.send("Starting to add the **Users** role to everyone (max 100 members)...")
+        message = await ctx.send("Starting to add the **Users** role to everyone (max 100 members)...")
+
         start = time.time()
         for member in ctx.guild.members:
             if member.bot:
                 nb_bot += 1
                 continue
-            has_role = False
-            for m_role in member.roles:
-                if m_role.name == 'Users':
-                    has_role = True
-                    break
+
+            has_role = discord.utils.get(member.roles, name="Users") is not None
             if has_role is True:
```

Comment:
this is redundant.   

if has_role:
    ## code

Reply:
redundant but pythonic. i'll change it